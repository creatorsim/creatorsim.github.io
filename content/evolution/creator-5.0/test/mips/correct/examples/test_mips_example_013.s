
#
# Creator (https://creatorsim.github.io/creator/)
#

.data
   utf8: .asciiz  "String in UTF-8: âˆ€ âˆ« âˆ‡ âˆˆ ðŸ ¬"

.text
main: li $v0, 1
      la $t0, utf8
loop: lb $a0, 0($t0)
      syscall
      addi $t0, $t0, 1
      bne $a0, $zero, loop
