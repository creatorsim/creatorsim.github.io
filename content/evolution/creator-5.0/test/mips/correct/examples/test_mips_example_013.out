
83
116
114
105
110
103
32
105
110
32
85
84
70
45
56
58
32
-30
-120
-128
32
-30
-120
-85
32
-30
-120
-121
32
-30
-120
-120
32
-16
-97
-96
-84
0
cr[PC]:0x20; ir[1,at]:0x200000; ir[2,v0]:0x1; ir[8,t0]:0x200026; keyboard[0x0]:''; display[0x0]:'83116114105110103321051103285847045565832-30-120-12832-30-120-8532-30-120-12132-30-120-12032-16-97-96-840'; 

